lib/src/virtual_host.dart

```diff
@@ -106,24 +106,24 @@ class _VirtualHost implements VirtualHost {
     }
     var controller = StreamController<HttpRequest>();
     var domains = host.split('.');
-    var current = _topDomain;
+    _VirtualHostDomain? current = _topDomain;
```

Comment:
Same comment here - add the non-null assertion at the assignment below (line 121)

Reply:
And done.